src/ui/components/Auth/Login.jsx

```diff
@@ -1,26 +1,47 @@
-import './auth.css';
-import React, { memo, useCallback, useRef, useState } from 'react';
+import React, { memo, useCallback, useState, useEffect } from 'react';
 import { TextField, Button, Snackbar } from '@material-ui/core';
 import { Alert } from '@material-ui/lab';
 import axios from 'axios';
 import { useQueryClient } from 'react-query';
+import useStyles from './styles';
 
 const Login = () => {
   const queryClient = useQueryClient();
-  const emailTextField = useRef();
-  const passwordTextField = useRef();
+  const classes = useStyles();
   const [message, setMessage] = useState(null);
-
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
   const handleClose = useCallback(() => setMessage(null));
 
+  useEffect(() => {
+    const listener = event => {
+      if (event.code === 'Enter' || event.code === 'NumpadEnter') {
+        event.preventDefault();
+        onSubmit();
+      }
+    };
+    document.addEventListener('keydown', listener);
+    return () => {
+      document.removeEventListener('keydown', listener);
+    };
+  }, [username, password]);
+
+  const _setUsername = event => {
```

Comment:
From my experience the underscore version is the one to not use. So the state would be defined as `[username, _setUsername] = ....` and then this would be `setUsername`. Also these two methods should be callbacks